clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10344,9 +10348,9 @@ void OffloadWrapper::ConstructJob(Compilation &C, const JobAction &JA,
 
     if (WrapperCompileEnabled) {
       // TODO Use TC.SelectTool().
-      ArgStringList ClangArgs{
-          TCArgs.MakeArgString("--target=" + TC.getAuxTriple()->str()), "-c",
-          "-o", Output.getFilename(), WrapperFileName};
+      ArgStringList ClangArgs{TCArgs.MakeArgString("--target=" + HostTripleStr),
+                              "-c", "-o", Output.getFilename(),
```

Comment:
There appears to be extra whitespaces added here and in L10353

Reply:
This was a formatting change suggested by `clang-format` when the `HostTripleStr` was updated.